automapper/src/main/java/com/valqueries/automapper/ValqueriesAccessDataLayerTestDouble.java

```diff
@@ -60,9 +62,10 @@ public CrudUpdateResult deleteById(K k) {
 
 	@Override
 	public CrudUpdateResult deleteByIds(Collection<K> collection) {
-		return () -> collection.stream()
-				.mapToInt(key -> getStore(modelType).remove(key) != null ? 1 : 0)
-				.sum();
+		List<T> existing = collection.stream()
+				.map(key -> getStore(modelType).remove(key))
```

Comment:
this looks weird, should it be filter instead of map? also can just return .count() on stream

Reply:
915cbe1 I was already working on this because it was wrong 😬